packages/contentful/migrations/0100-create-landing-page.cjs

```diff
@@ -0,0 +1,434 @@
+      {
+        unique: true
+      }
+    ])
+    .disabled(false)
+    .omitted(false);
+
+  imageCard
+    .createField('text')
+    .name('Text')
+    .type('Text')
+    .localized(true)
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  imageCard
+    .createField('image')
+    .name('Image')
+    .type('Link')
+    .localized(false)
+    .required(false)
+    .validations([
+      {
+        linkContentType: ['imageWithAttribution']
+      }
+    ])
+    .disabled(false)
+    .omitted(false)
+    .linkType('Entry');
+
+  // Create illustration group to be used by landing page
+  const illustrationGroup = migration
+    .createContentType('illustrationGroup')
+    .name('Illustration group')
+    .description('Section with text and grouped illustrations')
+    .displayField('name');
+
+  illustrationGroup
+    .createField('name')
+    .name('Name')
+    .type('Symbol')
+    .localized(true)
+    .required(true)
+    .validations([
+      {
+        unique: true
+      }
+    ])
+    .disabled(false)
+    .omitted(false);
+
+  illustrationGroup
+    .createField('text')
+    .name('Text')
+    .type('Text')
+    .localized(true)
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  illustrationGroup
+    .createField('hasPart')
+    .name('Sections')
+    .type('Array')
+    .localized(false)
+    .required(false)
+    .validations([
+      {
+        size: {
+          max: 6
+        }
+      }
+    ])
+    .disabled(false)
+    .omitted(false)
+    .items({
+      type: 'Link',
+
+      validations: [
+        {
+          linkContentType: ['illustration']
+        }
+      ],
+
+      linkType: 'Entry'
+    });
+
+  // Create image card group to be used by landing page
+  const imageCardGroup = migration
+    .createContentType('imageCardGroup')
+    .name('Image card group')
+    .description('Section with text and grouped image cards')
+    .displayField('name');
+
+  imageCardGroup
+    .createField('name')
+    .name('Name')
+    .type('Symbol')
+    .localized(true)
+    .required(true)
+    .validations([
+      {
+        unique: true
+      }
+    ])
+    .disabled(false)
+    .omitted(false);
+
+  imageCardGroup
+    .createField('text')
+    .name('Text')
+    .type('Text')
+    .localized(true)
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  imageCardGroup
+    .createField('hasPart')
+    .name('Sections')
+    .type('Array')
+    .localized(false)
+    .required(false)
+    .validations([
+      {
+        size: {
+          max: 6
+        }
+      }
+    ])
+    .disabled(false)
+    .omitted(false)
+    .items({
+      type: 'Link',
+
+      validations: [
+        {
+          linkContentType: ['imageCard']
+        }
+      ],
+
+      linkType: 'Entry'
+    });
+
+  // Create sub section to be used by landing page
+  const subSection = migration
+    .createContentType('subSection')
```

Comment:
since it only appears on landing pages should the naming reflect that?
`landingSubSection`?

Reply:
Updated name to `landingSubSection`